external/llvm-project/mlir/include/mlir/Dialect/GPU/GPUOps.td

```diff
@@ -1141,7 +1141,7 @@ def GPU_MubufLoadOp :
     GPU_Op<"buffer_load">,
     Arguments<(ins AnyMemRef:$memref,
                    Variadic<I32>:$indices)>,
-    Results<(outs AnyTypeOf<[I16, F16, F32,
+    Results<(outs AnyTypeOf<[I16, F16, F32, I32,
```

Comment:
could you help say the rationale for adding `I32`?
these relevant intrinsics don't really take LLVM i32 to my understanding.